lhotse/recipes/voxceleb.py

```diff
@@ -394,10 +427,8 @@ def _prepare_voxceleb_v2(
         recordings = []
         supervisions = []
         futures = []
-        for p in (corpus_path / split).glob("*.wav"):
-            futures.append(
-                ex.submit(_process_file, p, speaker_metadata, type="command")
-            )
+        for p in corpus_path.rglob("*.m4a"):
```

Comment:
Is the entire VoxCeleb2 distributed as m4a files?

Reply:
Ah, I think VoxCeleb1 is wav files, and 2 is m4a files. Is that right?

Reply:
correct